ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml

```diff
@@ -187,6 +187,29 @@ periodics:
       name: ""
       resources: {}
     serviceAccountName: ci-operator
+- agent: ""
+  cron: ""
+  decorate: true
+  interval: 24h
+  extra_refs:
+  - org: openshift
+    repo: openshift-azure
+    base_ref: master
+  name: periodic-ci-azure-vmimage-validate-osa311-latest
+  spec:
+    containers:
+    - args:
```

Comment:
I believe you need `make` as the first entry in `command`

Reply:
@stevekuznetsov sorry, I'm not sure that I understand what you mean. Should it be defined differently from the `periodic-ci-azure-vmimage` job above and other jobs?